pkg/apis/core/v1/helper/helpers.go

```diff
@@ -85,6 +85,21 @@ func HugePageSizeFromResourceName(name v1.ResourceName) (resource.Quantity, erro
 	return resource.ParseQuantity(pageSize)
 }
 
+// HugePageUnitSizeFromByteSize returns hugepage size has the format.
+// `size` must be guaranteed to divisible into the largest units that can be expressed.
+// <size><unit-prefix>B (1024 = "1KB", 1048576 = "1MB", etc).
+func HugePageUnitSizeFromByteSize(size int64) string {
```

Comment:
the only caller of this function appears to be kubelet.

Is there a reason we could not just use the utility we vendor in the kubelet already from runc?

Reply:
well, thinking on this more it is convenient to have this utility here since it parallels nicely with `HugePageSizeFromResourceName`

Reply:
runc cannot be used in helpers.go, because it is for linux.